.changeset/dull-spoons-clap.md

```diff
@@ -0,0 +1,8 @@
+---
+'@sap-ux-private/control-property-editor-common': minor
+'@sap-ux-private/preview-middleware-client': minor
+'@sap-ux/control-property-editor': minor
+'@sap-ux/preview-middleware': patch
```

Comment:
Thanks for adding the `preview- middleware`. Please make all `patch`. Sorry for not being clearer with my previous commit.

Reply:
Changeed in: https://github.com/SAP/open-ux-tools/pull/1422/commits/d04ce0cec87bb5338752c7eec2f487602623febe